google-summer-of-code/GSOC-STUDENT-TEMPLATE.md

```diff
@@ -39,8 +43,8 @@
 
 ### About me
 
-> Add your name, personal details and contact information (Slack ID, E-Mail). Optionally, add why you would you consider yourself as the best person for this project.
+> Introduce yourself here in a brief manner. Try to answer basic qustions like Who are you? What are your interest?. Add your name, personal details and contact information (Slack ID, E-Mail).
 
-### Appendix
+### What you expect from the mentor? (optional)
 
-> Mention any external references, information, hyperlinks here.
+> This field is optional. You can put your preference for what kind of mentorship will work the best for you. Anything that you want your mentor to have or follow. Anything regarding tech stack or experience can be put here.
```

Comment:
`This field is optional. You can choose what kind of mentorship will work best for you. You can put anything regarding technologies you might know or have experience in here.`

Reply:
ok